src/Process/3dTilesProcessing.js

```diff
@@ -96,7 +96,6 @@ export function pre3dTilesUpdate(context, layer) {
 function computeNodeSSE(camera, node) {
     if (node.boundingVolume.region) {
         const worldCoordinateCenter = node.boundingVolume.region.centerWorld;
-        worldCoordinateCenter.applyMatrix4(node.matrixWorld);
```

Comment:
I'd suggest you to use the same pattern as `TileProvider`:
- OBB expressed in tile local coords
- and add OBB as a tile's child (see TiledNodeProcessing also)